seqr/views/apis/summary_data_api_tests.py

```diff
@@ -413,6 +417,82 @@ def test_bulk_update_family_analysed_by(self, mock_load_uploaded_file):
         self.assertSetEqual({fab.created_by for fab in models}, {self.analyst_user})
         self.assertSetEqual({fab.family.family_id for fab in models}, {'1', '12'})
 
+        # Test AIP
+        aip_upload = {
+            'metadata': {
+                'categories': {
+                    '1': 'ClinVar Pathogenic',
+                    '2': 'New Gene-Disease Association',
+                    '3': 'High Impact Variant',
+                    '4': 'De-Novo',
+                    'support': 'High in Silico Scores'
+                }
+            },
+            'results': {
+                'HG00731': {
+                    '12-48367227-TC-T': {'categories': ['3', '4'], 'support_vars': ['2-103343353-GAGA-G']},
+                    '1-248367227-TC-T': {'categories': ['1'], 'support_vars': ['12-48367227-TC-T']},
+                },
+                'SAM_123': {
+                    '12-48367227-TC-T': {'categories': ['4', 'support'], 'support_vars': []},
+                },
+            }
+        }
+        mock_load_uploaded_file.return_value = aip_upload
```

Comment:
I noticed that single test functions include multiple test 'cases' (error cases, success cases). Is that a django standard or just how it's done in this repo?

Reply:
just this repo